addons/website_blog/controllers/main.py

```diff
@@ -63,12 +64,43 @@ def blogs(self, page=1, **post):
             page=page,
             step=self._blog_post_per_page,
         )
+        date_begin, date_end = post.get('date_begin'), post.get('date_end')
+        if date_begin and date_end:
+            domain += [("post_date", ">=", date_begin), ("post_date", "<=", date_end)]
+
+        # retrocompatibility to accept tag as slug
+        active_tag_ids = tag and [int(unslug(t)[1]) for t in tag.split(',')] or []
```

Comment:
`unslug ` was already there, I copied that part from  [main.py#L110](https://github.com/odoo/odoo/blob/master/addons/website_blog/controllers/main.py#L110)

Should I fix this in both places? Also, since this part is copy/pasted, should we create a common function to avoid killing kittens ([#r312465073](https://github.com/odoo/odoo/pull/34609/commits/12194d4aec2dc0fc878e42308db7a4792d6a95a6#r312465073)) ?

Reply:
after more anlayze... don't replace tags by tag to stay coherent with /blog/<>
and avoid to break SEO changing both